packages/flame/test/components/mixins/tappable_test.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:flame/components.dart';
+import 'package:flame/events.dart';
+import 'package:flame/game.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter/gestures.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Tappable', () {
```

Comment:
It would be good with some widget tests here too, similarly to:
https://github.com/flame-engine/flame/blob/main/packages/flame/test/events/flame_game_mixins/has_tappable_components_test.dart

Reply:
added.